tests/conftest.py

```diff
@@ -61,28 +61,30 @@ def _checked_call(self, cmd, expect_success=True, expect_error_output=False):
         outcome = 'succeeded' if proc.returncode == 0 else 'failed'
         allout = proc.stdout + proc.stderr
         if allout:
-            print(f'Client command {args} {outcome} with output:\n{allout}')
+            if log_output:
+                print(f'Client command {args} {outcome} with output:\n{allout}')
+            else:
+                print(f'Client command {args} {outcome} with output.')
```

Comment:
I found this confusing at first. How about printing `Client command {args} {outcome} (output suppressed)`?

Reply:
Fine for me.